app/controllers/api/v1/users_controller.rb

```diff
@@ -5,18 +5,27 @@ def show
     render json: user, status: 200
   end
 
+  def create
+    user = User.new(user_params)
+    if user.save
+      render json: user.id, status: 201
+    else
+      render json: { error: "An error occured" }, status: 400
```

Comment:
Depending on what's missing or if the email is not unique, we can eventually create a helper method to determine which error to render.

Reply:
Definitely! I want to beef up these methods once we get GraphQL implemented.